h2d/Graphics.hx

```diff
@@ -537,6 +537,31 @@ class Graphics extends Drawable {
 		lineTo(cx, cy);
 		flush();
 	}
+	
+	public function drawRectanglePie( cx : Float, cy : Float, width : Float, height : Float, angleStart:Float, angleLength:Float, nsegments = 0 ) {
+		if(Math.abs(angleLength) >= Math.PI*2) {
+			return drawRect(cx-(width/2), cy-(height/2), width, height);
+		}
+		flush();
+		lineTo(cx, cy);
+		if( nsegments == 0 )
+			nsegments = Math.ceil(Math.abs(width * angleLength / 4));
+		if( nsegments < 3 ) nsegments = 3;
+		var angle = angleLength / (nsegments - 1);
+		for( i in 0...nsegments ) {
+			var a = i * angle + angleStart;
+
+			var _width = Math.cos(a) * (width/2+1) * 1.42;

```

Comment:
Please avoid 1.42 unprecise constant, use (Math.PI * 0.5) instead

Reply:
Actually it is √2 (1.415) but I didn't want to calculate it everytime because of lacking performance.

But I can make it a constant like this:
`var squareRoot2 = 1.42;`

PI/2 = 1.57, and that makes a latency while changing the angle length.
`1.42` is the best I have experimented :D

Reply:
You can put it into a var outside of the loop :  var sq2 = Math.sqrt(2)